cli/js/unit_tests.ts

```diff
@@ -56,6 +57,8 @@ import "./url_test.ts";
 import "./url_search_params_test.ts";
 import "./utime_test.ts";
 import "./write_file_test.ts";
+
+// The ordering of these cannot be alphabetical
```

Comment:
?

Reply:
Sorry, I should have completed it... I went in there and got frustrated that they weren't all alphabetical, reordered them, and the permissions stuff has to come later, so I backed out my change, put half a comment there out of frustration, but didn't filly explain it.